### ISSUE TITLE
Discontinuing MultiMC on CurseForge

### ISSUE THREAD

**Issue by Madis0** [Fabulously-Optimized/fabulously-optimized#627]
Fabulously Optimized plans to discontinue distributing MultiMC versions on CurseForge, starting with v6.0.0.

# FAQ

## Why?

* The new CurseForge site layout is making it significantly harder for MultiMC and vanilla users to download there
* Due to #626, the version of MultiMC is already missing Sodium as of 4.8.1, and it is expected to miss more mods in the future
* These were never "server packs" but some users got confused
   * On the other hand, only "server packs" made them visible in the first place
* Removing that version simplifies the building and approval process on CurseForge
* MultiMC has now fixed the Modrinth issues with its latest 0.7 release
* Vanilla users are getting a #110 soon
* Pretty sure mod devs were never getting compensated when posted there
* Probably easier to make #462 work too
* Recently some approvals were unpredictably pushed for manual approval due to a Fabric Language Kotlin update (which is already updated on CF itself...)

## When?

MultiMC users [are right now no longer encouraged](https://wiki.download.fo/install-instructions#multimc) to download from CurseForge.

The deprecation itself will happen on Minecraft 1.21.

Edit: now vanilla instructions have been updated.

## What does it mean to users?

Awesome news! No, really:

* [MultiMC users will be able to download with a few clicks now](https://wiki.download.fo/install-instructions#multimc)
* [Vanilla users will be able to download with a few clicks as well](https://wiki.download.fo/install-instructions#minecraft-launcher-vanilla)
* Mods and FO get the download counts they deserve on Modrinth 
* No confusion about "server packs" because there never were any

## Really, no drawbacks at all?

Okay, there are some. But very minor:

* ~~Vanilla users will no longer be able to download FO _manually_. In terms of FO, that is actually still a good thing, as it means there will be less users who download and install just the mods, [which is a very wrong thing to do.](https://fabulously-optimized.gitbook.io/modpack/readme/vanilla-launcher-faq#so-i-just-copy-the-mods-right)~~ No longer the case 
* It may affect FO's download count and monetization on CurseForge a bit, but also maybe not at all
* ~~Pojav support may be slightly harder to do in the future, though not necessarily (if there is some kind of installer that can run on it)~~ Pojav now has a modpack installer

**Comment by Madis0** [comment]
While the new installer is not here yet, [I've made a new page](https://download.fo/vanilla) for vanilla user downloads, which downloads the pack and its mods directly from Modrinth and combines to a simple ZIP file.
[The install instructions have been updated](https://wiki.download.fo/readme/install-instructions#minecraft-launcher-vanilla) for 4.11.0+, with legacy ones available below.

CurseForge uploads for MMC Legacy will continue at least until FO 6.0.0-alpha.1.
